apps/ledger-live-desktop/src/renderer/families/internet_computer/ManageNeuronFlowModal/steps/ListNeuron.tsx

```diff
@@ -0,0 +1,168 @@
+import React, { useCallback, useEffect } from "react";
+import { useTranslation } from "react-i18next";
+import styled from "styled-components";
+import { SyncOneAccountOnMount } from "@ledgerhq/live-common/bridge/react/index";
+import TrackPage from "~/renderer/analytics/TrackPage";
+import Box from "~/renderer/components/Box";
+import BroadcastErrorDisclaimer from "~/renderer/components/BroadcastErrorDisclaimer";
+import ErrorDisplay from "~/renderer/components/ErrorDisplay";
+import { getAccountBridge } from "@ledgerhq/live-common/bridge/index";
+import WarnBox from "~/renderer/components/WarnBox";
+import List from "../../components/List";
+import { StepProps } from "../types";
+import Button from "~/renderer/components/Button";
+import { Trans } from "react-i18next";
+
+interface ContainerProps {
+  shouldSpace?: boolean;
+}
+
+const Container = styled(Box).attrs<ContainerProps>(() => ({
+  alignItems: "center",
+  grow: true,
+}))<ContainerProps>`
+  width: 100%;
+  justify-content: ${p => (p.shouldSpace ? "space-between" : "center")};
```

Comment:
same type of changed i recommended on Mina https://github.com/LedgerHQ/ledger-live/pull/12121/files#diff-3cce3bddfb22e08ca627fb48816df84ff91adbddd1107999fa0f7c42f0ecd766R27